### PR TITLE
 Basic Error 404 designed

### PR DESCRIPTION
**Pull request by EmilyDev**
Displays path that is giving 404
offers link to go home
offers link for a turtle

### CODE REVIEW

File: src/stylesheets/404.scss [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+      -webkit-animation: swing 1s ease;
+      animation: swing 1s ease;
+      -webkit-animation-iteration-count: 1;
+      animation-iteration-count: 1;
+    }
+    display: inline-flex;
+  }
+}
+
+h3{
```
Comment (EmilyDev) [inline src/stylesheets/404.scss]: ahh nice catch

File: app/components/home/NotFound.js [hunk 2]
```diff
@@ -1,14 +1,71 @@
...
+  let pageNotFound = window.location.href;
+  let response= ["This is embarrassing, isnt it?", "Oh boy, this is awkward.", "Something must have broke :(", " Oh Snap! We couldn't find the page", " Looks like you made a wrong turn.\n Dont worry! it happens to the best of us!"];
+
+  let index = Math.floor(Math.random() * (response.length - 1) + 1);
+
+  let count = 0;
+
+  let onclickHandler =()=> {
+
+    if (count <15){
```
Comment (jakepeyser) [inline app/components/home/NotFound.js]: why stop at 15? go big or go home

File: app/components/home/NotFound.js [hunk 3]
```diff
@@ -1,14 +1,71 @@
...
 
-)
+      <h3> {response[index]}</h3>
+      <p>{pageNotFound} was not available.</p>
+
+
+      <h2>{"Let's get back on track!"}</h2>
+    <div className="buttons">
+        <Link to="/">
+          <StepsFlatButton label="Home" onClick={"#"}/>
```
Comment (jakepeyser) [inline app/components/home/NotFound.js]: no `onClick` needed if you're linking

File: app/components/home/NotFound.js [hunk 4]
```diff
@@ -1,14 +1,71 @@
...
+          <g id="Page-1" stroke="none" strokeWidth="1" fill="none">
+          <g id="Group" transform="translate(0.000000, -0.362180)" fill="#FFFFFF">
+          <path d="M0,108.36218 L204,108.36218 L204,137.36218 L0,137.36218 L0,108.36218 Z M2,110.36218 L202,110.36218 L202,135.36218 L2,135.36218 L2,110.36218 Z" id="Shape"></path>
+          <path d="M50,54.36218 L204,54.36218 L204,83.36218 L50,83.36218 L50,54.36218 Z M52,56.36218 L202,56.36218 L202,81.36218 L52,81.36218 L52,56.36218 Z" id="Shape"></path>
+          <path d="M100,0.36218 L204,0.36218 L204,29.36218 L100,29.36218 L100,0.36218 Z M102,2.36218 L202,2.36218 L202,27.36218 L102,27.36218 L102,2.36218 Z" id="Shape"></path>
+          </g>
+        </g>
+      </svg>
+      </div>*/}
+      <div className="tank" id="tank" />
```
Comment (jakepeyser) [inline app/components/home/NotFound.js]: maybe add a bit of padding underneath the tank and bottom of the page? up to you

File: app/components/home/NotFound.js [hunk 5]
```diff
@@ -1,14 +1,71 @@
...
+    }
+
+  };
+
+  return(
   <div className="not-found">
-    <h1><p> Sorry, that page doesn't exist! </p></h1>
-    <h4><p> Please <Link to="/">click here</Link> to go home.</p></h4>
-  </div>
+    <Helmet title="404 NOT FOUND"/>
```
Comment (EmilyDev) [inline app/components/home/NotFound.js]: its okay the page is not aggressive >:D

**Pull request merged**